t/plugin/log-rotate.t

```diff
@@ -51,17 +51,30 @@ __DATA__
         content_by_lua_block {
             ngx.log(ngx.ERR, "start xxxxxx")
```

Comment:
The log here should be more meaningful, instead of writing a random

Reply:
got it